bin/build.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+declare version="3.1.0";
```

Comment:
This should be retrieved from package.json so that there's only one place the version needs to change.

Reply:
@XhmikosR You're too fast for me! ^^
Did it.